lessons/advanced/gen-stage.md

```diff
@@ -142,9 +142,9 @@ defmodule GenstageExample.ProducerConsumer  do
 end
 ```
 
-You may have noticed with our producer-consumer we've introduced a new function `handle_events/3` and option in `init/1`.  With the `subscribe_to` option we instruct GenStage to put us into communcation with a specific producer.
+You may have noticed with our producer-consumer we've introduced a new option in `init/1` and a new function: `handle_events/3`.  With the `subscribe_to` option, we instruct GenStage to put us into communcation with a specific producer.
 
-The `handle_events/3` method is our workhorse, where we receive our incoming events, process them, and return our transformed set.  As we'll see consumers are implemented in much the same way but the important difference is what our `handle_events/3` method returns and how it's used.   When we label our process a process-consumer the second argument of our tuple, `numbers` in our case, is used to meet the demand of consumers downstream.  In consumers this value is discarded.
+The `handle_events/3` method is our workhorse, where we receive our incoming events, process them, and return our transformed set.  As we'll see consumers are implemented in much the same way, but the important difference is what our `handle_events/3` method returns and how it's used.   When we label our process a process-consumer, the second argument of our tuple — `numbers` in our case — is used to meet the demand of consumers downstream.  In consumers this value is discarded.
```

Comment:
In this instance they're not hyphens, but rather [em dashes](https://en.wikipedia.org/wiki/Dash#Em_dash). 

> The em dash is used in several ways: primarily in places where a set of parentheses or a colon might otherwise be used,[30] it can show an abrupt change in thought or be used where a full stop (period) is too strong and a comma too weak. Em dashes are also used to set off summaries or definitions.

Some people like spaces around an em dash, some don't. (As you can see, I like spaces.) That's generally a style decision you make for your publication.

Anyway, I totally understand waiting for someone else to weigh in. Let me know if there's anything else I can help with.